src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -437,6 +437,90 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
     @SuppressWarnings({"Indentation", "unused"})
     private Object[] generateRepsNotificationScenarios() {
         return new Object[]{
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e", "8eb75404-a442-47aa-bab2-c4ba83a70900"),
+                "no",
+                "no",
+                "no",
+                "no",
+                "0",
+                "0",
+                "2"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Arrays.asList("7af36950-fc63-45d1-907d-f472fac7af06"),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "yes",
+                "no",
+                "1",
+                "0",
+                "1"
```

Comment:
According to the ticket, regardless of the subscription for the appellant, this letter should be sent to them.